crates/sdk/Cargo.toml

```diff
@@ -24,8 +24,8 @@ rustls = ["reqwest/rustls"]
 ndc-test = ["dep:ndc-test"]
 
 [dependencies]
-ndc-models = { git = "http://github.com/hasura/ndc-spec.git", tag = "v0.1.4" }
-ndc-test = { git = "http://github.com/hasura/ndc-spec.git", tag = "v0.1.4", optional = true }
+ndc-models = { git = "http://github.com/hasura/ndc-spec.git" }
```

Comment:
Note: `tag` temporarily removed so we can access `ndc_models::VERSION`, which is not yet in a tagged release.

Reply:
Would suggest you pin the version to specifically [`002923d5c337930b9c9007f05c768fe9a0575612`](https://github.com/hasura/ndc-spec/tree/002923d5c337930b9c9007f05c768fe9a0575612), otherwise your PR also brings in other ndc-models breaking changes, like all the newtypes.

Reply:
Done